### PR TITLE
OC-258_Migrate_microgrids_functionality_to_new_message_processing_solution

### PR DESCRIPTION
**Pull request by riannefijn**

### CODE REVIEW

File: osgp-protocol-adapter-iec61850/src/main/java/com/alliander/osgp/adapter/protocol/iec61850/application/config/MessagingConfig.java [hunk 1]
```diff
@@ -388,45 +130,20 @@ public OsgpRequestMessageSender osgpRequestMessageSender() {
...
-        redeliveryPolicy.setRedeliveryDelay(Long
-                .parseLong(this.environment.getRequiredProperty(PROPERTY_NAME_JMS_OSGP_RESPONSES_REDELIVERY_DELAY)));
-        redeliveryPolicy.setDestination(this.iec61850RequestsQueue());
-        redeliveryPolicy.setBackOffMultiplier(Double.parseDouble(
-                this.environment.getRequiredProperty(PROPERTY_NAME_JMS_OSGP_RESPONSES_BACK_OFF_MULTIPLIER)));
-        redeliveryPolicy.setUseExponentialBackOff(Boolean.parseBoolean(
-                this.environment.getRequiredProperty(PROPERTY_NAME_JMS_OSGP_RESPONSES_USE_EXPONENTIAL_BACK_OFF)));
-        return redeliveryPolicy;
+    public JmsConfiguration osgpResponseJmsConfiguration(final JmsConfigurationFactory jmsConfigurationFactory) {
+        return jmsConfigurationFactory.initializeConfiguration("jms.osgp.responses",
```
Comment (smvdheijden) [inline osgp-protocol-adapter-iec61850/src/main/java/com/alliander/osgp/adapter/protocol/iec61850/application/config/MessagingConfig.java]: No JMS Template needed here, initialize without JMS Template

File: osgp-protocol-adapter-iec61850/src/main/resources/osgp-adapter-protocol-iec61850.properties [hunk 2]
```diff
@@ -157,28 +142,15 @@ jms.iec61850.responses.use.exponential.back.off=true
 jms.iec61850.log.item.requests.queue=osgp.logging.protocol
 
 # --- DELIVERY OPTIONS ---
```
Comment (smvdheijden) [inline osgp-protocol-adapter-iec61850/src/main/resources/osgp-adapter-protocol-iec61850.properties]: Replace by default properties to reduce number of properties?

File: osgp-protocol-adapter-iec61850/src/main/resources/osgp-adapter-protocol-iec61850.properties [hunk 3]
```diff
@@ -126,25 +121,15 @@ jms.iec61850.requests.use.exponential.back.off=true
 jms.iec61850.responses.queue=osgp-core.1_0.protocol-iec61850.1_0.responses
 
 # --- DELIVERY OPTIONS ---
```
Comment (smvdheijden) [inline osgp-protocol-adapter-iec61850/src/main/resources/osgp-adapter-protocol-iec61850.properties]: Replace by default properties to reduce number of properties?

File: osgp-protocol-adapter-iec61850/src/main/resources/osgp-adapter-protocol-iec61850.properties [hunk 4]
```diff
@@ -188,25 +160,15 @@ jms.iec61850.log.item.requests.use.exponential.back.off=true
 jms.osgp.requests.queue=osgp-core.1_0.protocol-iec61850.1_0.requests 
 
 # --- DELIVERY OPTIONS ---
```
Comment (smvdheijden) [inline osgp-protocol-adapter-iec61850/src/main/resources/osgp-adapter-protocol-iec61850.properties]: Replace by default properties to reduce number of properties?

File: osgp-protocol-adapter-iec61850/src/main/resources/osgp-adapter-protocol-iec61850.properties [hunk 5]
```diff
@@ -108,15 +108,10 @@ jms.iec61850.requests.concurrent.consumers=2
 jms.iec61850.requests.max.concurrent.consumers=10
```
Comment (smvdheijden) [inline osgp-protocol-adapter-iec61850/src/main/resources/osgp-adapter-protocol-iec61850.properties]: Replace by default properties to reduce number of properties?

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://52.16.237.165:80/OSGP/Protocol-Adapter-IEC61850?pullRequest=82) <br/>All committers of the pull request should sign our Contributor License Agreement in order to get your pull request merged.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: riannefijn<br/>:x: web-flow

**Pull request merged**